tests/Integration/ApiPlatform/ModuleEndpointTest.php

```diff
@@ -301,6 +311,43 @@ public function restResetModuleNotActive(array $module): void
         self::assertResponseStatusCodeSame(400);
     }
 
+    public function testUninstallModule()
+    {
+        $module = [
+            'technicalName' => 'bankwire',
+            'deleteFile' => false
+        ];
+
+        // uninstall specific module deleteFile true
+        $bearerToken = $this->getBearerToken(['module_write']);
+        static::createClient()->request('PUT', sprintf('/module/%s/uninstall', $module['technicalName']), [
+            'auth_bearer' => $bearerToken,
+            'json' => [
+                'deleteFile' => $module['deleteFile'],
+            ],
+        ]);
+
+        self::assertResponseStatusCodeSame(204);
+
+    }
+
+    public function testBulkUninstallModule()
+    {
+        $modules = ['ps_featuredproducts', 'ps_emailsubscription'];
+
+        // uninstall specific module deleteFile true
+        $bearerToken = $this->getBearerToken(['module_write']);
+        static::createClient()->request('PUT', sprintf('/modules/uninstall'), [
+            'auth_bearer' => $bearerToken,
+            'json' => [
+                'modules' => $modules,
+                'deleteFile' => true,
+            ],
+        ]);
+
+        self::assertResponseStatusCodeSame(204);
+    }
```

Comment:
You should check that now the API returns a 404 error code when we fetch the module's infos, maybe it's worth adding a `checkModuleNotFound` method similar to `getModuleInfos` except it returns void and it checks for the 404 error code

This can be reused in other test methods